Dockerfile

```diff
@@ -20,9 +17,6 @@ ENV \
 RUN apt update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get install -y --no-install-recommends \
         ack \
-        git \
-        locales \
-        procps \
     && echo "en_US UTF-8" > /etc/locale.gen \
     && locale-gen en_US.UTF-8 \
     && echo 'export PS1="\\w\$ "' > /root/.bashrc \
```

Comment:
Is this still necessary?

Reply:
No, I guess we can just switch to the same as the core devcontainer and push zsh in 😉